schema/schema.go

```diff
@@ -77,23 +79,29 @@ func buildKindVersion(path, kind string, put *v3high.Operation, strict bool) (*K
 		if err != nil {
 			return nil, err
 		}
+
+		err = checkThatOrderArePresent(newKind)
+		if err != nil {
+			return nil, err
+		}
 	}
 	return newKind, nil
 }
 
 type tagParseResult struct {
 	kind    string
 	version int
+	order   int
 }
 
 func parseTag(tag string) (tagParseResult, error) {
 	// we extract kind and version from the tag
 	// e.g. cli_cluster_kafka_v1 -> kind: Cluster, version: 1
 	// e.g. cli_topic-policy_self-serve_v2 -> kind: TopicPolicy, version: 2
-	re := regexp.MustCompile(`cli_([^_]+)_[^_]+_v(\d+)`)
+	re := regexp.MustCompile(`cli_([^_]+)_[^_]+_v(\d+)(?:_(\d+))?`)
 	matches := re.FindStringSubmatch(tag)
 
-	if len(matches) < 3 {
+	if len(matches) < 4 {
```

Comment:
3 no? the last part is optional?

Reply:
Yes but it will be present in the array with an empty string